samples/wifi/sta/README.rst

```diff
@@ -0,0 +1,89 @@
+.. _wifi_station_sample:
+
+Wi-Fi: Station
+##############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to connect the Wi-Fi Station to a specified Access Point.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+This sample can perform Wi-Fi operations "Connect and Disconnect" in the 2.4GHz and 5GHz bands depending on the capabilities of an Access point.
+
+Using this sample, the development kit can connect to the specified Access Point :abbr:`STA (Station)` mode.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/wifi/sta`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Currently, the following board(s) are supported:
+
+* nRF7002 DK
+
+The below parameters should be configured in ``prj.conf``:
+
+  Ssid
```

Comment:
SSID, this is an acronym

Reply:
Also, you can use a bulleted list or a table with "param - description - example"